terraform/main.tf

```diff
@@ -36,7 +37,8 @@ module "enable_google_apis" {
   project_id                  = var.gcp_project_id
   disable_services_on_destroy = false
 
-  activate_apis = local.apis
+  # activate_apis is defined as base_apis and memorystore_apis if memorystore is enabled
```

Comment:
Praise: I am glad this dynamic API list creation works as we discussed! It makes for a much cleaner solution and far fewer TF resource blocks! Great job, Jasper!

Suggestion: The comment should be made much more generalized while still explaining what's being done. We should have not to update this for every variant we add.

Alternative: "`activate_apis` is the set of `base_apis` and the APIs required by user-configured deployment options."